src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -694,6 +721,32 @@ export class CellModel implements ICellModel {
 		return source;
 	}
 
+	// Create an iopub message to display either a display result or an error result,
+	// in order to be displayed as part of a cell's outputs
+	private toIOPubMessage(isError: boolean, message?: string): nb.IIOPubMessage {
```

Comment:
Could just have it take in the error object as optional as well and if it exists then print the error